evap/staff/tests/test_importers.py

```diff
@@ -612,14 +667,65 @@ def test_import_new_participant(self):
         success_messages, __, __ = PersonImporter.process_source_evaluation(
             ImportType.PARTICIPANT, self.evaluation1, test_run=True, source_evaluation=self.evaluation2
         )
-        self.assertIn("1 participants would be added to the evaluation", "".join(success_messages))
-        self.assertIn(f"{self.participant2.email}", "".join(success_messages))
+        self.assertIn("1 participant would be added to the evaluation", "".join(success_messages))
+        self.assertIn(str(self.participant2.email), "".join(success_messages))
 
         success_messages, __, __ = PersonImporter.process_source_evaluation(
             ImportType.PARTICIPANT, self.evaluation1, test_run=False, source_evaluation=self.evaluation2
         )
-        self.assertIn("1 participants added to the evaluation", "".join(success_messages))
-        self.assertIn(f"{self.participant2.email}", "".join(success_messages))
+        self.assertIn("1 participant added to the evaluation", "".join(success_messages))
+        self.assertIn(str(self.participant2.email), "".join(success_messages))
 
         self.assertEqual(self.evaluation1.participants.count(), 2)
         self.assertEqual(set(self.evaluation1.participants.all()), set([self.participant1, self.participant2]))
+
+    def test_person_success_messages(self):
+        success_messages, __, __ = PersonImporter.process_source_evaluation(
+            ImportType.PARTICIPANT, self.evaluation1, test_run=True, source_evaluation=self.evaluation3
+        )
+        self.assertIn("2 participants would be added to the evaluation", "".join(success_messages))
+        success_messages, __, __ = PersonImporter.process_source_evaluation(
+            ImportType.PARTICIPANT, self.evaluation1, test_run=False, source_evaluation=self.evaluation3
+        )
+        self.assertIn("2 participants added to the evaluation", "".join(success_messages))
+        success_messages, __, __ = PersonImporter.process_source_evaluation(
+            ImportType.PARTICIPANT, self.evaluation2, test_run=True, source_evaluation=self.evaluation3
+        )
+        self.assertIn("1 participant would be added to the evaluation", "".join(success_messages))
```

Comment:
Why? We just imported participants from evaluation3 to evaluation2, why does it say "1 would be added" now?